<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigForm</class>
 <widget class="QMainWindow" name="ConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/logo.png</normaloff>:/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Configurações</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Informações</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Data:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>67</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Hora:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="datePicker">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePicker">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Horarios de irrigação</string>
    </property>
    <widget class="QCheckBox" name="chkSeg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>92</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Seg</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkTer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>92</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ter</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkQui">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>92</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Qui</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkQua">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>92</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Qua</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkSab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>92</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sab</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkSex">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>92</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sex</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkDom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>92</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Dom</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerSeg1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerTer1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerQui1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerQua1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerSab1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerDom1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerSex1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>16</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerQua2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerSex2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerDom2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerQui2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerTer2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerSab2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timePickerSeg2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gravar Dados</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>btnExit</zorder>
   <zorder>btnSave</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
